docs/api.md

```diff
@@ -1301,6 +1309,10 @@ Fires when an attribute of an entity changes.
 #### "entityEquip" (entity)
 #### "entitySleep" (entity)
 #### "entitySpawn" (entity)
+#### "entityElytraFlew" (entity)
```

Comment:
we should explain what is entity or redirect to prismarine-entity

Reply:
It's already documented in the [entity section](https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#entity) of this document, so I'm not sure if I should do that

Reply:
no indeed don't